src/blocklyc.js

```diff
@@ -157,14 +157,6 @@ var graph_labels = null;
 var graph_csv_data = new Array;
 
 
```

Comment:
Removing individual globals

Reply:
var graph_csv_data = new Array; can be written as var graph_csv_data = []; Both a syntactically correct.